### PR TITLE
factor: Make the implementation of GCD more readable

### PR DESCRIPTION
**Pull request by nbraud**

### CODE REVIEW

File: src/uu/factor/src/numeric/gcd.rs [hunk 1]
```diff
@@ -9,39 +9,43 @@
...
     // Stein's binary GCD algorithm
     // Base cases: gcd(n, 0) = gcd(0, n) = n
-    if n == 0 {
-        return m;
-    } else if m == 0 {
-        return n;
+    if u == 0 {
+        return v;
+    } else if v == 0 {
+        return u;
```
Comment (sylvestre) [inline src/uu/factor/src/numeric/gcd.rs:18]:
```suggestion
    if u == 0 {
        return v;
    }
    if v == 0 {
        return u;
```

Comment (nbraud) [reply src/uu/factor/src/numeric/gcd.rs:18]: Any particular reason this is the preferred form?

Comment (sylvestre) [reply src/uu/factor/src/numeric/gcd.rs:18]: My Mozilla habits, the "else" is useless and makes it a tiny bit harder to read

Comment (nbraud) [reply src/uu/factor/src/numeric/gcd.rs:18]: *shrugs.* I find the reverse a tiny bit harder to read, but I also don't care to argue >_>'
Done

**Pull request merged**